Makefile

```diff
@@ -417,7 +417,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
 	rm -rf output/staging; \
-	VERSION="2.18"; \
+	VERSION="2.19"; \
```

Comment:
The current master branch (candidate build for May also) contain the version of files that I am asking to be placed here. However DSC has an older version of oms_configuraiton which overwrites the newer version thereby breaking the diagnostic channel logging capability.